HospitalWebApp/Controllers/EquipmentTypeController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using HealthcareBase.Service.HospitalResourcesService.EquipmentService;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using HealthcareBase.Dto;
+using HealthcareBase.Service.HospitalResourcesService.EquipmentService.Interface;
+using HospitalWebApp.Controllers.Interface;
+
+namespace HospitalWebApp.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class EquipmentTypeController : ControllerBase, IEquipmentTypeController
+    {
+        private readonly IEquipmentTypeService _equipmentTypeService;
+
+        public EquipmentTypeController(EquipmentTypeService equipmentTypeService)
+        {
+            _equipmentTypeService = equipmentTypeService;
+        }
+
+        [HttpGet]
+        [Route("GetAll")]
+        public IActionResult GetAllEquipmentTypes()
```

Comment:
All paths starts with lowercase, you should follow that convention.

Reply:
Fixed.